lib/ws_servers/api/algos/algo_worker.js

```diff
@@ -47,7 +47,7 @@ class AlgoWorker {
       withHeartbeat: true,
       affiliateCode,
       wsURL,
-      restURL
```

Comment:
restURL is used by `AOHost` .. it shouldn't be removed

Reply:
fixed